sync-core/src/test/java/com/cloudant/http/HttpTest.java

```diff
@@ -119,4 +122,52 @@ public void testReadBeforeExecute() throws IOException {
         Assert.assertEquals(bis.available(), data.getBytes().length);
         client.deleteDb();
     }
+
+
+    //NOTE: This test doesn't work with specified couch servers,
```

Comment:
So it's pointless running it. Can you think of a way of skipping it if it isn't relevant, using one of the `Assume` methods?